Client/src/Components/Search/ISearchDatasetsFormModel.tsx

```diff
@@ -1,21 +1,35 @@
+import * as Yup from 'yup'
+
+// export interface ISearchDatasetsFormModel {
+//   oxidizer: string,
+//   year: string,
+//   outputFormat: string,
+//   categories: string,
+//   subcategories: string,
+//   fuel: string,
+//   author: string,
+//   diluent: string,
+// }
+
 export interface ISearchDatasetsFormModel {
-  oxidizer: string,
-  year: string,
-  outputFormat: string,
-  categories: string,
-  subcategories: string,
-  fuel: string,
-  author: string,
-  diluent: string,
+  datasetId?: number[],
```

Comment:
Are these supposed to be optional?

Reply:
yes, actually all of them should be optional, because querying should be supported for each single field alone, gonna fix